tests/python/contrib/test_cudnn.py

```diff
@@ -72,6 +72,67 @@ def verify():
 
     verify()
 
```

Comment:
can you modify this verify() function for cudnn 2D to make sure that output is correct? You can use [conv2d_nchw_python](https://github.com/apache/incubator-tvm/blob/master/topi/python/topi/testing/conv2d_nchw_python.py#L87) function.